### ISSUE TITLE
Calibrator doesn't wait for user input when calibrating Kpwm

### ISSUE THREAD

**Issue by nunoguedelha** [robotology-playground/sensors-calib-inertial#7]
### Context
Issue occurred on commit 09d1263, while testing the motor Kpwm calibration.

User just confirmed to proceed after the friction parameters fitting completed. Following steps should have followed:
1. Motor parameter Kpwm calibration (left leg joint 4: left knee)
a. joint encoder and torque data acquisition
    i.  promt the user for pushing the leg during the data acquisition
    ii. run data acquisition while waiting for user confirmation to stop
--> these two last steps are missing.

**Comment by nunoguedelha** [comment]
The feature is missing in the initial design. Fix ongoing

**Comment by nunoguedelha** [comment]
Fixed by commit f880ccc89f2e735db492d624ca79069de66bafeb.
```
Add prompt definition in the sequence profile configuration for friction calibration:

If the defined prompt string is empty, no specific action is taken in the motion sequencer 
run method. Otherwise, the sequencer step is run (setEncoders) without waiting for motion 
completion, and the user is prompt with the defined prompt string.
```
